tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestClassInfoRunClassCleanupIntegration.cs

```diff
@@ -35,14 +35,8 @@ public static class TestClassInfoRunClassCleanupIntegration
     /// <param name="arg">Instance of the argument</param>
     /// <returns>Calltarget state value</returns>
     internal static CallTargetState OnMethodBegin<TTarget, TArg>(TTarget instance, TArg arg)
-        where TTarget : ITestAssemblyInfo
     {
-        if (!MsTestIntegration.IsEnabled)
-        {
-            return CallTargetState.GetDefault();
-        }
-
-        if (TestClassInfoRunClassInitializeIntegration.TestClassInfos.TryGetValue(instance.Instance, out var suiteObject) && suiteObject is TestSuite suite)
+        if (MsTestIntegration.IsEnabled && TestClassInfoRunClassInitializeIntegration.TestClassInfos.TryGetValue(instance, out var suiteObject) && suiteObject is TestSuite suite)
```

Comment:
Is this integration method still called after the change in `TestClassInfoRunClassInitializeIntegration.cs`?

Reply:
Yes, this is still relevant, the modification in the other class was about changing the field we use to inject the delegate for cleanup.